src/components/tree/treeViewFunctions.js

```diff
@@ -273,6 +273,7 @@ export const salientPropChanges = (props, nextProps, tree) => {
   const branchThickness = props.tree.branchThicknessVersion !== nextProps.tree.branchThicknessVersion;
   const layout = props.layout !== nextProps.layout;
   const distanceMeasure = props.distanceMeasure !== nextProps.distanceMeasure;
+  const rerenderAllElements = nextProps.quickdraw === false && props.quickdraw === true;
```

Comment:
I had been imagining this as running `rerenderAllElements` from a particular action that fires when toggling from `quickdraw=true` to `quickdraw=false`. I agree with this setup however.